contrib/parseq-lambda-names/src/main/java/com/linkedin/parseq/lambda/LambdaClassLocator.java

```diff
@@ -0,0 +1,37 @@
+package com.linkedin.parseq.lambda;
+
+import org.objectweb.asm.ClassVisitor;
+
+
+/**
+ * This class visits all the classes and locates classes generated for Lambda expressions.
```

Comment:
This comment is not very clear to me e.g. what does it mean that it visits all classes.